CHANGELOG.md

```diff
@@ -3,6 +3,99 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.18.0
+
+_Sep 20, 2024_
+
+We'd like to offer a big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Support [Row spanning](https://mui.com/x/react-data-grid/row-spanning/) on the Data Grid that automatically merges the consecutive cells in a column based on the cell value
+
+  <img width="600" src="https://github.com/user-attachments/assets/d32ec936-d238-4c92-9e1a-af6788d74cdf" alt="data grid row spanning" />
```

Comment:
Adding a height would improve the UX.

```suggestion
  <img width="600" height="326" src="https://github.com/user-attachments/assets/d32ec936-d238-4c92-9e1a-af6788d74cdf" alt="data grid row spanning" />
```

so the content doesn't move while reading the changelog after the video loads.

---

On the video itself, I think that correct scale + x2 pixel density + higher fps would improve the UX. See the guide in https://www.notion.so/mui-org/High-quality-media-5a1686758f4f4267989b67a1c2c270da?pvs=4#03d3a56e4f3e4890be21a6d56a005194. Here is how it feel for comparison:

![before](https://github.com/user-attachments/assets/d32ec936-d238-4c92-9e1a-af6788d74cdf)

![after](https://github.com/user-attachments/assets/03960387-0cc2-44da-9c6a-6c438a45fa50)

Reply:
Cool, thanks for linking the page, it definitely looks better.